src/lib/rules/strict-transport-security/strict-transport-security.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * @fileoverview This rule checkes header strict-transport-security is sent for resources served over HTTPS, and max-age has proper  value.
+ */
+
+import { RuleContext } from '../../rule-context'; // eslint-disable-line no-unused-vars
+import { isDataURI, isHTTPS, normalizeString } from '../../utils/misc'; // eslint-disable-line no-unused-vars
+import { IAsyncHTMLElement, IResponse, IFetchEnd, IRule, IRuleBuilder } from '../../types'; // eslint-disable-line no-unused-vars
+
+// ------------------------------------------------------------------------------
+// Public
+// ------------------------------------------------------------------------------
+
+const rule: IRuleBuilder = {
+    create(context: RuleContext): IRule {
+        /** The minimum period (in seconds) allowed for `max-age`. */
+        let minimum: number;
+
+        const loadRuleConfigs = () => {
+            minimum = (context.ruleOptions && context.ruleOptions.minimum) || 10886400;
+        };
+
+        const maxAgeLessThan = (headerValue: string, limit: number) => {
+            const maxAge = headerValue.match(/max-age=([^;]+);|max-age=([^;]+)$/).pop();
```

Comment:
@alrra I found it here: https://tools.ietf.org/html/rfc6797#section-8.1
`If a UA receives more than one STS header field in an HTTP
      response message over secure transport, then the UA MUST process
      only the first such header field.`
> Question: What do browser do if you have multiple max-age directives? Do they follow the spec?